config/manager/controller_manager_config.template.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: eraser.sh/v1
+kind: EraserConfig
+health:
+  healthProbeBindAddress: :8081
+metrics:
+  bindAddress: 127.0.0.1:8080
+webhook:
```

Comment:
why is there a webhook here?

Reply:
These were the defaults from `main.go`, they can most likely be removed.

Reply:
done (removed)